### PR TITLE
Orchestrator first flow Testing + Refactor + BUG FIX

### PR DESCRIPTION
**Pull request by Timothylock**
Good news! 

The flow now works all the way up to ` Locate IR Points` (payload hasn't provided us metadata to test correctly) 

This PR is still missing 
image

Will be added on in a later PR. Speaking of which, @levmckinney can you tell me where those 2 modules are?

### CODE REVIEW

**Comment by je888** [comment]
"'./tmp/json/'" does not exist

**Review by an-bluecat — APPROVED** [review]
BEAUTIFUL

File: src/task1/filter_module/detect.py [hunk 1]
```diff
@@ -71,12 +72,18 @@ def run(input_dir, output_dir):
             rel_out_path = None
             if output_dir is not None:
                 rel_out_path = os.path.join(output_dir, f)
-            if rel_out_path.lower().endswith(IMAGE_FORMATS):
+
+            if not rel_path.lower().endswith(IMAGE_FORMATS):
+                continue
+
+            try:
```
Comment (Timothylock) [inline src/task1/filter_module/detect.py:79]: 🙈 The goal is to try-catch the entire program

Comment (Timothylock) [reply src/task1/filter_module/detect.py:79]: -helps the program recover gracefully from a corrupted img file-

File: src/task1/orchestrator/mainWindowLayout.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 from PyQt5 import QtWidgets
 
+LEFT_LABEL_POS = 240
```
Comment (Timothylock) [inline src/task1/orchestrator/mainWindowLayout.py:3]: HEY MA, CONSTANTS!

File: src/task1/orchestrator/session.py [hunk 3]
```diff
@@ -13,6 +13,12 @@ def startupCheck(self, jsondir):
     :return:
     """
 
+    # If the json directory doesn't exist, make it and continue starting orchestrator
```
Comment (Timothylock) [inline src/task1/orchestrator/session.py:16]: Orchestrator depends on the folder existing

File: src/task1/orchestrator/README.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ This is the orchestration GUI that will be running all of the modules for task 1
 
 The orchestrator will unlock buttons for different modules depending on where the last part of the workflow that has been completed.
 
+*THE ORCHESTRATOR MUST BE RUN FROM THIS FOLDER (do not for example do `python3 ./orchestrator/orchestrator.py`)*
```
Comment (Timothylock) [inline src/task1/orchestrator/README.md:8]: Its the users fault now

**Review by levmckinney — APPROVED** [review]
Looks good  to me!

**Pull request merged**